### PR TITLE
[REVIEW] Update build script.

### PR DESCRIPTION
**Pull request by avolkov1**
The build script creates a docker container with greenflow, greenflowlab and corresponding plugins currently residing in the repo.

### CODE REVIEW

File: gQuant/docker/build.sh [hunk 1]
```diff
@@ -61,54 +71,96 @@ case $CUDA_VERSION in
...
-WORKDIR /home/quant/greenflow
-ENTRYPOINT MODULEPATH=\$HOME/greenflow/modules jupyter-lab --allow-root --ip=0.0.0.0 --no-browser --NotebookApp.token=''
+
+## install greenflow plugins
+ADD --chown=$USERID:$USERGID ./build/plugins /home/quant/plugins
+RUN cd /home/quant/plugins/gquant_plugin && pip install .
+RUN cd /home/quant/plugins/dask_plugin && pip install .
+
+WORKDIR /home/quant
+ENTRYPOINT MODULEPATH=\$HOME/plugins/gquant_plugin/modules jupyter-lab \
```
Comment (yidong72) [inline gQuant/docker/build.sh:149]: If we start the jupyter-lab at the /home/quant directory, the notebooks won't properly.
Change the path to 
```python
WORKDIR /home/quant/plugins/gquant_plugin
ENTRYPOINT MODULEPATH=\$HOME/plugins/gquant_plugin/modules jupyter-lab \
  --allow-root --ip=0.0.0.0 --no-browser --NotebookApp.token='' \
  --ContentsManager.allow_hidden=True
```

**Review by yidong72 — CHANGES_REQUESTED** [review]
need to fix the path

File: gQuant/docker/build.sh (hunk 1 above)
Comment (avolkov1) [reply gQuant/docker/build.sh:149]: Fixed in commit 563ddde.

**Comment by yidong72** [comment]
Looks good. merge the change

**Pull request merged**